### PR TITLE
Copy to side ui

### PR DESCRIPTION
**Pull request by jakearchibald**
Trying a different button position. What do you think?

screen shot 2018-10-28 at 11 37 56

### CODE REVIEW

**Comment by jakearchibald** [comment]
Preview URL https://deploy-preview-229--squoosh.netlify.com/

**Comment by mustafa-x** [comment]
can we add an undo function? so if someone copies settings across, they can undo them

**Comment by jakearchibald** [comment]
The technical side of this is easy, but I'm not sure what the UI would be.

**Comment by jakearchibald** [comment]
Maybe we could have a snackbar that says "Settings copied" with an undo button

**Comment by mustafa-x** [comment]
We should a pill that comes up when a user undoes or redoes something. That animates then disappears. (see bottom left|)

example

**Comment by mustafa-x** [comment]
Also I think separating the buttons download and copy across would work better.

**Comment by mustafa-x** [comment]
I was commenting on the first design :)

**Comment by developit** [comment]
undo notification could be a toast with 500ms timeout.

**Comment by jakearchibald** [comment]
@developit exactly that in https://github.com/GoogleChromeLabs/squoosh/pull/233 😄

**Comment by jakearchibald** [comment]
(well, I think it's a few seconds rather than 500ms)

**Comment by mustafa-x** [comment]
screen shot 2018-10-30 at 1 36 44 pm

Might be worth disabling the button if both sides are the same, or after it has been duplicated.

**Comment by mustafa-x** [comment]
I was reviewing https://deploy-preview-233--squoosh.netlify.com/

**Comment by jakearchibald** [comment]
Cool. Those changes are over in https://github.com/GoogleChromeLabs/squoosh/pull/233.

The "233" in the URL refers to the pull request number.

**Comment by jakearchibald** [comment]
@mustafa-x you can get to the result of a PR in the "Show all checks" section:

screen shot 2018-10-30 at 13 55 56

File: src/components/results/style.scss [hunk 1]
```diff
@@ -16,31 +16,55 @@
 
 .results {
   display: grid;
-  grid-template-columns: 1fr auto;
+  grid-template-columns: [text] 1fr [copy-button] auto [download-button] auto;
```
Comment (surma) [inline src/components/results/style.scss]: YEAAAAH, grid featuerzzz 😎

**Pull request merged**